.github/workflows/publish.yml

```diff
@@ -13,9 +13,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
+      - uses: actions/setup-java@v2
         with:
-          java-version: 1.8
+          distribution: 'adopt'
+          java-version: '11'
```

Comment:
any reason to not bump up to 16 or 17?

Reply:
for now, consistency with the CI jobs as those also use 11